server/routes/serviceProviderReferrals/showReferralPresenter.test.ts

```diff
@@ -18,7 +18,6 @@ describe(ShowReferralPresenter, () => {
   const cohortIntervention = interventionFactory.build({
     contractType: { code: 'PWB', name: 'Personal wellbeing' },
     serviceCategories: cohortServiceCategories,
-    serviceCategory: cohortServiceCategories[0],
   })
 
   const referralParams = {
```

Comment:
_random question_  _not a blocker_
Can the `referral.serviceCategoryId` in this also go?

Reply:
yes.. although we also need to remove it in the DraftReferralDTO on the back-end too

Reply:
Not sure if UI does it first if that causes issues